vowpalwabbit/test_red.cc

```diff
@@ -485,12 +562,17 @@ void learn_automl(tr_data& data, multi_learner& base, multi_ex& ec)
     }
   }
 
+  data.cm.gen_configs(data.oc, ec);
```

Comment:
this should be inside one_step, where appropriate. Feel free to update the interface of one_step to include whatever you need to forward

Reply:
keep oracle inside tr_data for now, but lets see where it falls later

Reply:
Updated to do this step in one_step only when Experimenting. This is also requires that one_step takes two arguments